conf/deployment/vsphere/upi_1az_rhcos_multus_public_cluster_vsan_3m_3w_ipv6.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Can you change file name?

conf/deployment/vsphere/upi_1az_rhcos_multus_vsan_3m_3w.yaml
https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/blob/master/jobs/qe_ci_production_job_triggers.groovy?ref_type=heads#L76

Reply:
It's a new deployment so it can't be in Jenkins yet